app/services/hyrax/custom_queries/navigators/parent_works_navigator.rb

```diff
@@ -0,0 +1,33 @@
+# Navigate from a resource to the child works in the resource.
+module Hyrax
+  module CustomQueries
+    module Navigators
+      class ParentWorksNavigator
+        # Define the queries that can be fulfilled by this navigator.
+        def self.queries
+          [:find_parent_works, :find_parent_work_ids]
+        end
+
+        attr_reader :query_service
+
+        def initialize(query_service:)
+          @query_service = query_service
+        end
+
+        # Find parent works of a given resource, and map to Valkyrie Resources
+        # @param [Valkyrie::Resource]
+        # @return [Array<Valkyrie::Resource>]
+        def find_parent_works(resource:)
+          query_service.find_inverse_references_by(resource: resource, property: :member_ids).select(&:work?)
```

Comment:
who is implementing `#work?` here? i only see this method defined in `Wings`-specific behavior. if that's right, we can't count on it being available here